### DISCUSSION TITLE
Backup configuration

### DISCUSSION THREAD

**Discussion by adevoss** [inspircd/inspircd#1995] [Support]
What files do I have to backup for when something happens to the box and I have to restore the inspircd instance?
Of course `/etc/inspircd` and `/etc/default/inspircd` but
where are the registered channels/users stored?

Creating a disk image is a solution but takes a lot of disk space.

**Comment by genius3000** [comment]
That should be it for InspIRCd configuration backup. Registered users and channels are handled in whichever services package you're using (most common ones being Anope or Atheme). I'd assume system installs would have their configuration and saved data either in `/etc` or `/var/lib`.
fwiw, the general setup I do for remote backups is to capture all of `/home` and `/etc`. That works for most of my servers and containers, saving the necessary configuration only.

**Reply by adevoss** [reply]
> (most common ones being Anope or Atheme)

Just installed inspIRCd to setup an IRC server. To install I only installed the .deb package so I don't know.
Do I have to install Anope/Atheme to register channels/users?
Any big differences between the two?

**Comment by sadiepowell** [comment]
The two important directories you will need to back up using our packages are `/etc/inspircd` (config files) and `/var/lib/inspircd` (data files).

You may also wish to back up `/var/log/inspircd` if you care about log files.

**Reply by adevoss** [reply]
I have installed Anope. If I understand it that's handling the permanent channels/users (and more).
An option is to store data in a flatfile (to keep it simple for now).

That's was missing in my setup. Now I have to figure out how to configure it.
Thanks.
